x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -52,4 +53,15 @@ export const uiSettings: Record<string, UiSettingsParams<boolean | number>> = {
     }),
     schema: schema.boolean(),
   },
+  [enableInfrastructureView]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.enableInfrastructureView', {
+      defaultMessage: 'Infrastructure feature',
+    }),
+    value: true,
+    description: i18n.translate('xpack.observability.enableInfrastructureViewDescription', {
+      defaultMessage: 'Enable the Infrastruture view feature on APM UI',
```

Comment:
"on APM UI" sounds somewhat off. @formgeist @bmorelli25 what do you think? The same wording is used for the comparison feature right above this one.

Reply:
@dgieselaar Agreed, that seems like an old reference. 

```suggestion
      defaultMessage: 'Enable the Infrastruture view feature in APM app',
```